CMakeLists.txt

```diff
@@ -534,78 +539,26 @@ if(WITH_RADOSGW_BEAST_FRONTEND)
 	list(APPEND BOOST_COMPONENTS coroutine context)
 endif()
 
-if (WITH_SYSTEM_BOOST)
+set(Boost_USE_MULTITHREADED ON)
+# require minimally the bundled version
+if(WITH_SYSTEM_BOOST)
   if(ENABLE_SHARED)
     set(Boost_USE_STATIC_LIBS OFF)
   else()
     set(Boost_USE_STATIC_LIBS ON)
   endif()
+  find_package(Boost 1.61 COMPONENTS ${BOOST_COMPONENTS} REQUIRED)
```

Comment:
is there a reason this is staying at 1.61, when `BuildBoost.cmake` and the submodule are at 1.63?